xdsl/context.py

```diff
@@ -19,6 +19,9 @@ class Context:
     _loaded_attrs: dict[str, type[Attribute]] = field(
         default_factory=dict[str, type[Attribute]]
     )
+    _loaded_types: dict[str, type[Attribute]] = field(
```

Comment:
```suggestion
    _loaded_types: dict[str, type[TypeAttribute]] = field(
```
Is this worth doing?

Reply:
I was thinking of it, and I personally have no clue. I could add that if you think it's worth